docs/reference/commandline/run.md

```diff
@@ -582,12 +550,14 @@ still store what's been written to `STDERR` and `STDOUT`.
 $ cat somefile | docker run -i -a stdin mybuilder dobuild
 ```
 
-This is how piping a file into a container could be done for a build.
+This is a way of using `--attach` to pipe a build file into a container.
 The container's ID will be printed after the build is done and the build
 logs could be retrieved using `docker logs`. This is
 useful if you need to pipe a file or something else into a container and
 retrieve the container's ID once the container has finished running.
 
+See also [the `docker cp` command](/engine/reference/commandline/cp/).
```

Comment:
We may need to either use a full (`docs.docker.com/....`) URL for this if we want the link to work on GitHub, or a relative URL (`cp.md`)